src/components/DisplayArticle.jsx

```diff
@@ -1,18 +1,46 @@
 import { useParams } from "react-router-dom";
 import { getArticle } from "../utils/api";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { increaseVotes } from "../utils/api";
 
 const DisplayArticle = () => {
+  const clickHandler = (e, increment) => {
+    setVotes((currVotes) => {
+      const newVotes = currVotes + increment;
+      return newVotes;
+    });
+    increaseVotes(article_id, increment).then((res) => {});
+  };
+
   const { article_id } = useParams();
   const [selectedArticle, setSelectedArticle] = useState({});
-  getArticle(article_id).then((article) => {
-    setSelectedArticle(article);
-  });
+  const [votes, setVotes] = useState(selectedArticle.votes);
+
+  useEffect(() => {
+    getArticle(article_id).then((article) => {
+      setSelectedArticle(article);
+    });
+  }, [votes]);
   return (
     <div>
       <h2>{selectedArticle.title}</h2>
       <p>{selectedArticle.body}</p>
       <h3>{selectedArticle.author}</h3>
+      <p>votes: {selectedArticle.votes}</p>
```

Comment:
We would want to render you votes in state here, rather than the article votes! This is most likely what is causing the delay